js/main.js

```diff
@@ -29,6 +29,8 @@ function formSave(event) {
     // Reset the form
     $entryImage.setAttribute('src', 'images/placeholder-image-square.jpg');
     $form.reset();
+    const $deleteButton = document.querySelector('button.delete');
+    $deleteButton.classList.add('hidden');
```

Comment:
I would move the `querySelector` outside the function. By including it here, you're having to traverse the DOM over and over again every time this function gets called, but if we move it to the global space, we only need to do that search once, and we'll always have a reference to it. Especially because you query for this element in other places as well, so you can just reuse the same variable